### PR TITLE
objbench: fix ceph panic when put empty file

### PR DESCRIPTION
**Pull request by zhijian-pro**
image

### CODE REVIEW

File: pkg/object/ceph.go [hunk 1]
```diff
@@ -139,6 +139,9 @@ func (c *ceph) Put(key string, in io.Reader) error {
 		if b, ok := in.(*bytes.Reader); ok {
 			v := reflect.ValueOf(b)
 			data := v.Elem().Field(0).Bytes()
+			if len(data) == 0 {
+				return errors.New("ceph: can't put empty file")
+			}
 			return ctx.WriteFull(key, data)
 		}
 		buf := cephPool.Get().([]byte)
```
Comment (davies) [inline pkg/object/ceph.go:147]: `in` could also be empty string, we should return an error

Comment (zhijian-pro) [reply pkg/object/ceph.go:147]: `ctx.WriteFull` will panic when  `len(data)=0`

Comment (zhijian-pro) [reply pkg/object/ceph.go:147]: image

Comment (davies) [reply pkg/object/ceph.go:147]: I meant we should also fix the case that it's not `*bytes.Reader`

**Pull request merged**